### ISSUE TITLE
About refinement of labels

### ISSUE THREAD

**Issue by strongzero** [DeepLabCut/DeepLabCut#517]
When we refine labels if right-click on the label, we can delete a label, but once a label is deleted, it cannot be retrieved.

My request is to enable to place the labels once again if we delete it by mistake.
I'm using turtle and they often put their hands and foots in their shells, so when I refine labels I have to delete the labels many times and labels overlap each other many times so I sometimes make a mistake that I delete wrong labels.
If I make this mistake, I have to refine all over again. In some case, more than 100 frames.

I would appreciate it if you could consider this request.

**Comment by MMathisLab** [comment]
But that is precisely why we have it ask you “are you sure you want to delete this point?” - due to how this is saved, it’s not straightforward to retrieve. 

You *can* also then open the image up in the labeling GUI if you need to place back the label.

**Comment by MMathisLab** [comment]
I would also strongly urge you to start with a more diverse training set vs refining hundreds of labels; as you see this is more time consuming that just starting with a diverse set of frames to start, sampled across different videos, backgrounds, turtles, etc. hope that helps.

**Comment by strongzero** [comment]
Hi Mackenzie,

Thank you for your advice!
As you said, that solution might be the best way.
I'll try to use more diverse training data sets.

Best regards,
Shohei
